index.bs

```diff
@@ -3184,23 +3183,23 @@ Each {{WebRequestEvent}} has:
                               |formData|[|entry|[0]].
 
 
-                  1. Set |details|'s {{RequestBody/formData}} to |formData|.
+                  1. Set |details|["{{RequestBody/formData}}"] to |formData|.
 
-          1. Set |details|'s {{WebRequestBeforeRequestDetails/requestBody}}
+          1. Set |details|["{{WebRequestBeforeRequestDetails/requestBody}}"]
```

Comment:
more for my personal interest: when do you use <variable name>'s ... and when do you use <variable name>[...]. is it dictionary vs struct?

Reply:
`|struct|'s {{Struct/member}}` vs `|dictionary|["{{Dictionary/member}}"]`

I had flipped these around in a few places, which is what this PR fixes.